cloud-agent/service/server/src/main/scala/org/hyperledger/identus/agent/server/MainApp.scala

```diff
@@ -139,6 +144,38 @@ object MainApp extends ZIOAppDefault {
       |""".stripMargin)
         .ignore
 
+      appConfig <- ZIO.service[AppConfig].provide(SystemModule.configLayer)
+      // these services are added to any DID document by default when they are created.
+      defaultDidDocumentServices = Set(
+        DidService(
+          id = appConfig.agent.httpEndpoint.serviceName,
+          serviceEndpoint = DidServiceEndpoint
+            .Single(
+              DidServiceEndpoint.UriOrJsonEndpoint
+                .Uri(
```

Comment:
Moving this behind `AppConfig` and call `appConfig.defaultDidDocumenteService` here might be a good idea.